source/change-log/index.html.md.erb

```diff
@@ -0,0 +1,47 @@
+---
+title: Change log
+weight: 50
+description: You need to send header data to HMRC when you use our APIs. Check the fraud prevention specification. Check the formatting requirements and how to contact us about missing data.
+---
+<%= partial 'tracking-consent/partials/google_tag_manager_js' %>
+
+# Change log
```

Comment:
Upgraded the `Change log` header to `h1` but kept versions at `h3` so they don't appear in left hand nav.